docs/morevp.md

```diff
@@ -342,6 +341,43 @@ Bob’s exit will have priority of position of `UTXO1`.
 11. Bob receives the value of `UTXO2` first, Operator receives the value of `UTXO4` second (ideally contract is empty by this point).
 All bonds are refunded.
 
+### Operator tries to steal funds from a multi-input in-flight transaction.
+
+1. Alice spends `UTXO1a`, Malory spends `UTXO1m` in `TX1` to Bob, creating `UTXO2`.
+2. `TX1` is in-flight.
+3. Operator creates invalid deposit, creating `UTXO3`.
+4. Operator spends `UTXO3` in `TX3`, creating `UTXO4`.
+5. Operator starts an exit referencing `TX3` and places `exit bond`.
```

Comment:
Regarding `in-flight` exit - operator needs to prove inclusion of each input in the chain. Have you had in mind that `UTXO3` is an output created in transaction spending mentioned invalid deposit (not a deposit itself). I think this is some mental shortcut used here - as operator could not prove inclusion of `UTXO3` therefore his exit is not accepted (:question:)

Reply:
Oh and the previous case shares the same drawback

Reply:
I think by "invalid deposit", a transaction that looks like this is meant:
```
inputs: []
outputs: [{operator, token, bazilion} <~~~~UTXO3]
```
it is included by the operator in a child chain block and its inclusion can be proven.

Implementation-wise it is not a deposit, but it protocol-wise it is - a transaction with no inputs.

As a consequence, `TX3` is a "valid" transaction, from point of view of IFE

Reply:
> I think by "invalid deposit", a transaction `inputs: []`

I believe it's still not possible to start `in-flight` exit from `TX3` because it contradicts the `exitability` property then:

> transaction can be called “exitable” if the transaction is correctly formed (e.g. more input value than output value,

Reply:
Oh, no we're talking about the same thing:
`[] - tx2 -> UTXO3 - tx3 -> UTXO4`
I got it now... :smiley_cat: